doozer/doozerlib/cli/rpms.py

```diff
@@ -302,3 +311,58 @@ async def _build_rpm(runtime: Runtime, builder: RPMBuilder, rpm: RPMMetadata):
             record["task_url"] = task_urls[0]
         runtime.record_logger.add_record(action, **record)
     return record["status"]
+
+
+async def update_konflux_db(runtime, rpm: RPMMetadata, record: dict):
+    nvrs = record["nvrs"].split(",")
+
+    with runtime.shared_koji_client_session() as koji_api:
+        builds = get_build_objects(nvrs, koji_api)
+
+    _, version, _ = await exectools.cmd_gather_async(
+        ["rpmspec", "-q", "--qf", "%{version}", "--srpm", "--undefine", "dist", "--",
```

Comment:
Curious if there is a reason to get version this way vs parsing it via nvr?

Reply:
Extracting `4.18.0` from `ose-aws-ecr-image-credential-provider-4.18.0-202409182058.p0.g876afc6.assembly.test.el8` looked error prone, NVR-dependent, and not very readable in the code.

Reply:
thinking if there a lot of builds, it might take a long time to run a command for each of them. Also we regularly use `rpm_utils.parse_nvr(nvr)` for a lot of things, so i think it's pretty reliable.

Reply:
Nice, done in 5eaf5cedc6610a5d390f6e38b4b2da7b5771e1fa